src/bci_build/package/postgres/README.md.j2

```diff
@@ -0,0 +1,114 @@
+# {{ image.pretty_name }} Container Image
+
+## Description
+
+PostgreSQL, also known as Postgres, is a relational database management system (RDBMS) with an emphasis on extensibility and SQL compliance.
+
+PostgreSQL implements the majority of the SQL:2011 standard, is ACID-compliant and transactional (including most DDL statements) avoiding locking issues using multiversion concurrency control (MVCC), provides immunity to dirty reads and full serializability; handles complex SQL queries using many indexing methods that are not available in other databases; has updateable views and materialized views, triggers, foreign keys; supports functions and stored procedures, and other expandability, and has a large number of extensions written by third parties.
+
+## Usage
+
+By default, the image launches PostgreSQL with the same configuration that comes with SUSE Linux Enterprise Server.
+
+The only environment variable required to start the container is the PostgreSQL root password.
+
+```ShellSession
+$ podman run -it --rm -p 5432:5432 -e POSTGRES_PASSWORD=my-password -v /path/to/data:/var/lib/pgsql/data {{ image.pretty_reference }}
+```
+
+**Note:** The path `/var/lib/pgsql/data` is a local volume. When mounting a volume to `/var/lib/pgsql` the data from `/var/lib/pgsql/data` is not persisted on the mounted volume unless you explicitly mount it.
```

Comment:
I'm trying to parse that sentence and it doesn't work for me. what are you saying? if you bind mount something over it like given above, the volume will not be used?

Reply:
yes, I can remove this bit but in theory, someone could mount the parent folder.

Reply:
How about converting it to a generic "Volumes" section like we have in other container image readmes?

Reply:
yes, that is a better solution.

Reply:
I removed this bit and added a new volume section.